src/formatDate.js

```diff
@@ -50,7 +50,34 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSpliter = fromFormat[3];
+  const newSpliter = toFormat[3];
+  const time = {};
+  const rezult = [];
+
+  const oldData = date.split(oldSpliter);
+
+  for (let i = 0; i < 3; i++) {
+    time[fromFormat[i]] = oldData[i];
+  }
+
+  if (time.YY) {
+    if (time.YY < 30) {
```

Comment:
What does it mean 30, 20, 19?  Looks like magic numbers. Move it to variables with meaningful names